internal/controllers/operator_controller.go

```diff
@@ -107,6 +107,12 @@ func checkForUnexpectedFieldChange(a, b operatorsv1alpha1.Operator) bool {
 }
 
 // Helper function to do the actual reconcile
+//
+// Today we always return ctrl.Result{} and an error.
+// But in the future we might update this function
+// to return different results (e.g. requeue).
+//
+//nolint:unparam
```

Comment:
Same as in rukpak (see [this thread](https://github.com/operator-framework/rukpak/pull/635#discussion_r1229859464)).